src/composables/trade/useValidation.ts

```diff
@@ -1,6 +1,17 @@
 import { computed } from 'vue';
 import useAuth from '@/composables/useAuth';
-import i18n from '@/plugins/i18n';
+import { ETHER } from '@/constants/tokenlists';
+
+const MIN_ETH_REQUIRED = 0.0001;
+
+export enum TradeValidation {
+  VALID,
+  NO_ACCOUNT,
+  EMPTY,
+  NO_ETHER,
+  NO_BALANCE,
+  NO_LIQUIDITY
+}
```

Comment:
Added enum for better separation of concerns (composable doesn't have to know about localization) and type safety.